tests/unit/mocks.py

```diff
@@ -108,11 +114,15 @@ def __init__(
 
 
 async def instance_metadata_success(*args: Any, **kwargs: Any) -> MockMetadata:
-    return MockMetadata(datetime.datetime.utcnow() + datetime.timedelta(minutes=10))
+    return MockMetadata(
+        datetime.datetime.now(datetime.timezone.utc) + datetime.timedelta(minutes=10)
```

Comment:
What is the significance of 10? Ditto below.

Reply:
No real significance, just a trivial time in the future for tests to pass.